source/agora/test/EnrollmentManager.d

```diff
@@ -73,5 +73,5 @@ unittest
 
     // check if nodes contains enrollment data previously sent.
     nodes.each!(node =>
-        retryFor(node.hasEnrollment(enroll.utxo_key) == true, 5.seconds));
+        retryFor(node.hasEnrollment(enroll.utxo_key) == true, 10.seconds));
```

Comment:
Why?

Reply:
It's changed temporally before requesting a review in order to find the reason why the timeout occured in the Github build.